src/config.rs

```diff
@@ -160,19 +167,21 @@ fn parse_cli_args() -> Result<CliArgs, getopts::Fail> {
 impl Config {
     /// Create a new configuration for creating Verso instance. It must provide the path of
     /// resources directory.
-    pub fn new(resource_dir: PathBuf) -> Self {
+    pub fn new() -> Self {
         let mut opts = default_opts();
-        let args = parse_cli_args().unwrap_or_default();
+        let mut args = parse_cli_args().unwrap_or_default();
 
         if let Some(devtools_port) = args.devtools_port {
             opts.devtools_server_enabled = true;
             opts.devtools_port = devtools_port;
         }
 
+        let resource_dir = args.resource_dir.take().unwrap_or(resources_dir_path());
```

Comment:
We currently don't use this cli arg elsewhere but we might in the future, since this clone is trivial, I think it's better to just keep the `args` real to the real command line args

```suggestion
        let resource_dir = args.resource_dir.clone().unwrap_or_else(|| resources_dir_path());
```